lib/watir/elements/element.rb

```diff
@@ -213,26 +213,28 @@ def drag_and_drop_by(right_by, down_by)
     end
 
     #
-    # Flashes (change background color far a moment) element.
+    # Flashes (change background color to a new color and back a few times) element.
     #
-    # @example
+    # @examples
     #   browser.text_field(name: "new_user_first_name").flash
+    #   browser.text_field(name: "new_user_first_name").flash("green",3,0.05)
     #
 
-    def flash
+    def flash(color="yellow",flashes=5,delay=0.1)
```

Comment:
Let's use keyword arguments so that you can pass just one of them (e.g. you only want to control number of flashes):

```ruby
def flash(color: 'yellow', flashes: 5, delay: 0.1)
  ...
end
```